Intersect.Client/Entities/Entity.cs

```diff
@@ -1386,6 +1386,147 @@ public virtual void DrawName(Color textColor, Color borderColor = null, Color ba
             );
         }
 
+        // Tags
+        public virtual void DrawTag(NpcTagPos npcTagPos, PlayerTagPos playerTagPos)
```

Comment:
There are a lot of different variable declarations in this method that can/should be using `var` instead of explicit type names.

Reply:
👍🏽 

Entity DrawTag (Npc):
`var tagPos = Options.Npc.TagPosition;`

Player Override:
`var tagPos = Options.Player.TagPosition;`